scm/driver/gitlab/testdata/webhooks/pipeline_hook.json.golden

```diff
@@ -0,0 +1,50 @@
+{
+  "repo": {
+    "id": "1",
+    "namespace": "gitlab-org",
+    "name": "gitlab-test",
+    "clone": "http://192.168.64.1:3005/gitlab-org/gitlab-test.git",
+    "cloneSSH": "git@192.168.64.1:gitlab-org/gitlab-test.git",
+    "link": "http://192.168.64.1:3005/gitlab-org/gitlab-test",
+    "branch": "master",
+    "private": false
+  },
+  "commit": {
+    "sha": "bcbb5ec396a2c0f828686f14fac9b80b780504f2",
+    "message": "test\n",
+    "author": {
+      "name": "User",
+      "email": "user@gitlab.com"
+    },
+    "committer": {
+      "name": "User",
+      "email": "user@gitlab.com"
+    },
+    "link": "http://example.com/gitlab-org/gitlab-test/commit/bcbb5ec396a2c0f828686f14fac9b80b780504f2"
+  },
+  "Execution": {
+    "id": "31",
```

Comment:
is this the correct id?  this looks like the auto-incremented execution number vs the identifier stored in the database.  I can't tell which id field is correct (build id, job id, etc)

Reply:
@bradrydzewski this is the execution id `http://example.com/gitlab-org/gitlab-test/-/pipelines/31`